pysmt/smtlib/script.py

```diff
@@ -194,10 +209,23 @@ def add_command(self, command):
     def evaluate(self, solver):
         log = []
         for cmd in self.commands:
-            r = evaluate_command(cmd, solver)
+            r = omt_evaluate_command(cmd, solver)
             log.append((cmd.name, r))
+
         return log
 
+    def evaluate(self, solver, client_data):
```

Comment:
In python there is no function overload! You cannot have two functions with the same name: only the second one is considered (it "overrides" the first one). I am sure your IDE gives you a warning about this, **please read the warnings**!

Moreover, do not change the old behavior, add new behavior as needed: in this case you can keep the old `evaluate` function and simply add a new one `omt_evaluate`...